cache-manager.php

```diff
@@ -246,11 +248,65 @@ function queue_post_purge( $post_id ) {
 		}
 
 		$this->purge_urls[] = get_permalink( $post_id );
+	}
+	
+	function transition_post_status( $new, $old, $post ) {
+		if ( $this->site_cache_purged ) {
+			return;
+		}
+		
+		if ( defined( 'WP_IMPORTING' ) ) {
+			return;
+		}
+		
+		// Only send PURGE requests for public post types
+		if ( ! is_post_type_viewable( $post->post_type ) ) {
+			return;
+		}
+		
+		// Don't PURGE if we're not transitioning to or from publish
+		if ( $new === $old || ( $new != 'publish' && $old != 'publish' ) ) {
```

Comment:
I thought about that... I think publish is always publish. Edit Flow adds intermediate steps before publish, but (afaik) the publish state is still the same.